The tests is stochastical and has been seen failing in EasyBuild.
Just increase the tolerance to catch larg(er) differences

author: Alexander Grund (TU Dresden)
diff -ur openmm-8.0.0/plugins/drude/tests/TestDrudeNoseHoover.h orig/plugins/drude/tests/TestDrudeNoseHoover.h
--- openmm-8.0.0/plugins/drude/tests/TestDrudeNoseHoover.h	2026-09-17 09:57:48.631105218 +0200
+++ orig/plugins/drude/tests/TestDrudeNoseHoover.h	2026-09-17 09:55:41.345637744 +0200
@@ -289,8 +289,8 @@
     }
     double comTemperature = (2*comKineticEnergy / (nDoF*BOLTZ));
     double relTemperature = (2*relKineticEnergy / (nDoF*BOLTZ));
-    ASSERT_USUALLY_EQUAL_TOL(targetTemperature, comTemperature, 0.01);
-    ASSERT_USUALLY_EQUAL_TOL(drudeTemperature, relTemperature, 0.01);
+    ASSERT_USUALLY_EQUAL_TOL(targetTemperature, comTemperature, 0.1);
+    ASSERT_USUALLY_EQUAL_TOL(drudeTemperature, relTemperature, 0.1);
 }
 
 void setupKernels(int argc, char* argv[]);
